packages/vaadin-grid/src/vaadin-grid-keyboard-navigation-mixin.js

```diff
@@ -346,9 +346,9 @@ export const KeyboardNavigationMixin = (superClass) =>
     _parseEventPath(path) {
       const tableIndex = path.indexOf(this.$.table);
       return {
-        rowGroup: path[tableIndex - 1],
-        row: path[tableIndex - 2],
-        cell: path[tableIndex - 3]
+        rowGroup: tableIndex >= 3 ? path[tableIndex - 1] : null,
+        row: tableIndex >= 3 ? path[tableIndex - 2] : null,
+        cell: tableIndex >= 3 ? path[tableIndex - 3] : null
```

Comment:
`tableIndex >= 3` is repeated 3 times. Could improve this by extracting it into a variable which would also make the code more readable, because the name of the variable indicates why in this case there is no cell/row/rowGroup.